src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -995,7 +995,14 @@ internal override void AfterAddingTypeMembersChecks(ConversionsBase conversions,
 
             foreach (var parameter in this.Parameters)
             {
-                parameter.Type.CheckAllConstraints(conversions, parameter.Locations[0], diagnostics);
+                if (parameter.Type.IsTupleType)
+                {
+                    parameter.Type.TupleUnderlyingType.CheckAllConstraints(conversions, parameter.Locations[0], diagnostics);
```

Comment:
>CheckAllConstraints [](start = 55, length = 19)

I think it would be better if CheckAllConstraints handled tuple type symbols itself.